inspirehep/base/templates/format/record/Inspire_Default_HTML_brief_macros.tpl

```diff
@@ -32,12 +36,57 @@
   {% if record.get('publication_info')|is_list() %}
     {% for pub_info in record.get('publication_info')%}
       {% if pub_info.get('journal_title') and pub_info.get('journal_volume') and  pub_info.get('year') and pub_info.get('page_artid') %}
-      <span class="text-left"><i>{{ pub_info.get('journal_title') }}</i> {{ pub_info.get('journal_volume') }} ({{pub_info.get('year')}}) {{ pub_info.get('page_artid') }}</span><br/>
+        {% if loop.first %}
+          Published in
+        {% endif %}
+        {% if not loop.first %}
+          and
+        {% endif %}
+        <i>{{ pub_info.get('journal_title') }}</i> {{ pub_info.get('journal_volume') }} ({{pub_info.get('year')}}) {{ pub_info.get('page_artid') }}
+      {% endif %}
+    {% endfor %}
+  {% else %}
+    {% if record.get('publication_info').get('journal_title') and record.get('publication_info').get('journal_volume') and  record.get('publication_info').get('year') and record.get('publication_info').get('page_artid') %}
```

Comment:
Here you are doing some different if `record.get('publication_info')` is a list or not.

In the first case you show any part of the publication info available but in the second only if all the parts are there you show the publication info. This should be consistent in both places.

And the information shown should reflect the same way we do it nowadays in INSPIRE - see https://github.com/inspirehep/inspire/blob/master/bibformat/format_elements/bfe_INSPIRE_publi_info.py#L28

Can you create a separate issue for this? We will merge this PR first and then you address it.